.github/workflows/test.yml

```diff
@@ -26,8 +26,11 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip setuptools wheel "coverage[toml]" pytest pytest-asyncio pytest-cov pytest-mock
-          pip install -U -r requirements.txt
+          python -m pip install --upgrade "coverage[toml]" pytest pytest-asyncio pytest-cov pytest-mock
+
+      - name: Install package
+        run: |
+          pip install -e .
```

Comment:
You can (should?) use single line strings here

Reply:
Either work; would you like me to change it?

Reply:
I'm not really someone that can decide on such things, but a normal string would look nicer here

Reply:
It seems all of the other steps use this format

Reply:
I agree that it's better as a single line string but the repo uses multiline strings for all other `run`s including single line ones so it makes sense to keep it consistent